doc/nrf/libraries/modem/location.rst

```diff
@@ -204,11 +204,32 @@ Use GNSS and cellular and set custom timeout values for them:
    location_config_defaults_set(&config, ARRAY_SIZE(methods), methods);
 
    /* Now you have default values set and here you can modify the parameters you want */
+   config.timeout = 180 * MSEC_PER_SEC;
    config.methods[0].gnss.timeout = 90 * MSEC_PER_SEC;
    config.methods[1].cellular.timeout = 15 * MSEC_PER_SEC;
 
    err = location_request(&config);
 
+Use method priority list defined by Kconfig options and set custom timeout values for entire location_request() and cellular positioning:
```

Comment:
If location_request() is a function, use the following tagging: :c:func:`function_name`